.bazelrc

```diff
@@ -21,12 +21,15 @@ build:clang --action_env=CXX=clang++
 build:asan-libfuzzer --config=clang
 build:asan-libfuzzer --linkopt=-fsanitize=fuzzer,address
 build:asan-libfuzzer --copt=-fsanitize=fuzzer,address
+build:asan-libfuzzer --//fuzzing:engine=libfuzzer
 
 # Flags for Clang with MSAN and libfuzzer
 build:msan-libfuzzer --config=clang
 build:msan-libfuzzer --linkopt=-fsanitize=memory,fuzzer
 build:msan-libfuzzer --copt=-fsanitize=memory,fuzzer
+build:asan-libfuzzer --//fuzzing:engine=libfuzzer
```

Comment:
Typo: Should be `msan-libfuzzer`.

The fact that no test fails because of this is a sign we don't have enough testing :) Feel free to file an issue for this.

Reply:
Created #61 for this issue.